cmd/eks-a-tool/cmd/cloudstackrmvms.go

```diff
@@ -0,0 +1,68 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"github.com/aws/eks-anywhere/pkg/providers/cloudstack/decoder"
+	"log"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/aws/eks-anywhere/pkg/executables"
+	"github.com/aws/eks-anywhere/pkg/filewriter"
+	"github.com/aws/eks-anywhere/pkg/validations"
+)
+
+const dryRunFlag = "dry-run"
+
+var cloudstackRmVmsCmd = &cobra.Command{
+	Use:    "vms <cluster-name>",
```

Comment:
rmvms?

Reply:
no. This is on top of the previous cobra command which specifies the "rm". The full command is `eks-a-tool cloudstack rm vms` and it's split across cobra files.

Reply:
```
➜  eks-anywhere git:(cloudstack-rm-vms) ✗ bin/eks-a-tool cloudstack rm vms -h
This command removes vms associated with a cluster name

Usage:
  eks-a-tool cloudstack rm vms <cluster-name> [flags]

Flags:
      --dry-run   Dry run flag
  -h, --help      help for vms

Global Flags:
  -v, --verbosity int   Set the log level verbosity
```